umm-spec-lib/src/cmr/umm_spec/test/dif10_expected_conversion.clj

```diff
@@ -337,22 +336,10 @@
   (when-let [media (first media)]
     [media]))
 
-(defn- expand-file-dist-infos
-  "Parse each format to see if the format contains multiple values. If it does
-   then replace the distribution info that exists and add new ones for each
-   multple format that exists."
-  [file-dist-info]
-  (let [formats (get (select-keys file-dist-info [:Format]) :Format)
-        parsed-format (format-util/parse-distribution-formats formats)]
-    (for [[index value] (map-indexed vector parsed-format)]
-      (if (= index 0)
-        (assoc file-dist-info :Format value)
-        (conj (assoc file-dist-info :Format value))))))
-
 (defn- expected-file-dist-info
   "Created expected FileDistributionInformation for ArchiveAndDistributionInformation map."
   [file-dist-infos]
-  (when-let [file-dist-infos (flatten (map expand-file-dist-infos file-dist-infos))]
+  (when-let [file-dist-infos (flatten (map conversion-util/expand-file-dist-infos file-dist-infos))]
```

Comment:
You should be able to replace `(flatten (map` with `(mapcat`.

Reply:
I was. thanks